continuous-testing/use-cases/markmap.md

```diff
@@ -1,56 +1,115 @@
 * Continuous Testing
     * For Development APIs
-        * Unit Testing
-            * xUnit (e.g. Junit\[Java\], PyUnit\[Python\], NUnit\[C#\], CppUnit\[C++\]
-            * Tools
-                * Java
-                * Python
-                * C#
-                * C/C++
-        * Integration, Dependency and Stub Mocking
-            * Simulate dependencies
-            * Stub out coupled services for planned results
-            * Mock specific application behavior
-            * Tools
-                * Java
-                    * Mockito
-                    * EasyMock
-                    * JMock
-                * Python
-                    * 
-                * C#
-                * C/C++
-        * Code Quality Analysis
-            * Semmle
+        * For module testing (xUnit)
+            * (Java) [JUnit](https://junit.org/junit5/) [&#x1F3C1; STARTER KIT](../starter-kits/#unit-teting-with-java)
```

Comment:
Same as Integration Guide suggestions, please use a proper prefix to make it more readable. e.g. "Using (Java) JUnit...". This applies to all the nodes.

Reply:
There's only a single item here which explains the prefix. Based on yesterday's conversation, extraneous branches (single line from node-to-node) were removed to economize the presentation. Would it be preferred to do it like this?
```
For module testing --o-- Using Java --o-- Using JUnit
```

Reply:
Hey @jpl-jengelke - that's a great question, personally I think its fine to just leave it the way you have it now, something like:
`Using (Java) [JUnit](https://junit.org/junit5/) [&#x1F3C1; STARTER KIT]`.

Your example of using a branch for `Using Java --o-- Using JUnit` probably only makes sense if we have multiple Java tools recommended down-the-line.

Reply:
Thanks. I think we are good to go then.